linera-core/src/unit_tests/wasm_worker_tests.rs

```diff
@@ -296,6 +298,7 @@ where
             events: vec![Vec::new()],
             state_hash: creator_state.crypto_hash().await?,
             oracle_responses: vec![Vec::new()],
+            operation_results: vec![vec![0x0F, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00].into()],
```

Comment:
Is this the current counter value?

```suggestion
            operation_results: vec![bcs::to_bytes(15u64)?.into()],
```

Reply:
Sure as you wish. As its just a constant.